about.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>confrince</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <nav class="contact-nav">
```

Comment:
- [ ] [OPTIONAL] I put this in optional because it was missed by the other code reviewers, but you need to follow the original layout, your nav content is way too wide, you need to add some margins/paddings, check the difference

Your design:

![image](https://user-images.githubusercontent.com/60631456/124291736-7d22d700-db1a-11eb-9dc0-576c7f6c80a7.png)

Original design:

![image](https://user-images.githubusercontent.com/60631456/124291814-962b8800-db1a-11eb-8602-8f9901a23278.png)